src/components/RelatedVideo/RelatedVideos.component.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+const RelatedVideo = styled(Link)`
+  display: grid;
+  grid-template-columns: 1fr 2fr;
+  padding-left: 5px;
+  color: black;
+`;
+
+function RelatedVideos({ id }) {
+  const [ytRelatedVideos, setYtRelatedVideos] = useState(null);
```

Comment:
Is null a good default value? Consider the API failing. Is your code robust enough to handle that case? Consider an empty array, data object `[]`